### ISSUE TITLE
2FA setup 

### ISSUE THREAD

**Issue by GoTeslaGo** [lwitzani/homebridgeStatusWidget#2]
I have 2FA setup on my homebridge so it’s not connecting. Is there a parameter / setting to allow the script to work with 2FA?

**Comment by lwitzani** [comment]
So you have 2FA to access the browser page provided by the app https://github.com/oznu/homebridge-config-ui-x ? 
2FA is not considered atm in my widget.

**Comment by GoTeslaGo** [comment]
Yup setup for browser access. Will you be able to add this feature at some point?

**Comment by lwitzani** [comment]
On the page https://github.com/oznu/homebridge-config-ui-x i don‘t read anything about 2FA. How do you even enable it? I think i won‘t support it in my widget. Sorry. Maybe you can do it yourself :)

**Comment by GoTeslaGo** [comment]
It’s an option from the web front end for homebridge config ui

I wish I could do it but I’m not a coder!

**Comment by lwitzani** [comment]
Is your config ui accessable from outside of your network? You could deactivate 2FA if it is not that important to you(?). Where can i find the option for 2FA in the config Ui?

**Comment by GoTeslaGo** [comment]
its within the User Accounts menu of Homebridge

**Comment by lwitzani** [comment]
haha now i finally found it. There is really a 2FA setting! But i will probably not integrate it :) sorry. Will let you know if i change my mind!

**Comment by xcytre** [comment]
I set up a non-admin account without 2fa to use for the widget.
